src/openrct2-ui/scripting/CustomWindow.cpp

```diff
@@ -49,6 +49,24 @@ namespace OpenRCT2::Ui::Windows
         { WWT_RESIZE, 1, 0, 0, 14, 0, 0xFFFFFFFF, STR_NONE },                // content panel
     };
 
+// TODO make a function that maps this.
+    static constexpr std::array<std::string_view, 99999> WindowClassifications = {
```

Comment:
Why creating a 100k sized array if you know exactly how many elements it contains?

If you don't want to count, use an `std::vector` instead. In any case, I don't see how this relates to the changeset.

Reply:
I know, I just couldn't be bothered to count them at the time, WIP